src/hotspot/share/gc/g1/g1CollectedHeap.hpp

```diff
@@ -1161,7 +1161,6 @@ class G1CollectedHeap : public CollectedHeap {
   void par_iterate_regions_array_part_from(HeapRegionClosure* cl,
```

Comment:
The comment needs to be updated as well. `const uint* regions` can be written as `const uint regions[]` to make the concept clearer, IMO.

Reply:
I have modified the function name too.